package.json

```diff
@@ -6,7 +6,18 @@
   "keywords": [
     "qooxdoo",
     "compiler",
-    "cli"
+    "cli",
+    "framework",
+    "front-end",
+    "mobile",
+    "back-end",
```

Comment:
backend (vs back-end) and frontend are common spellings. Maybe add them without the dashes as well?

Reply:
The main NPM keywords are actually with the dashes, but sure, we can add the other spelling as well.